developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md

```diff
@@ -0,0 +1,310 @@
+      ```
+
+    - For the `genesis_file_location` key, provide the full path to the `genesis.blob` file. For example:
+
+      ```
+      genesis_file_location: "/path/to/aptos-core/genesis.blob"
+      ```
+
+9. Start your local FullNode by running the below command:
+
+  ```
+  cargo run -p aptos-node --release -- -f ./public_full_node.yaml
+  ```
+
+You have now successfully configured and started running a FullNode connected to Aptos devnet.
+
+:::note
+
+This will build a release binary: `aptos-core/target/release/aptos-node`. The release binaries tend to be substantially faster than debug binaries but lack debugging information useful for development. To build a debug binary, omit the `--release` flag.
+
+:::
+
+### Using Docker
+
+This section describes how to configure and run your FullNode using Docker.
+
+1. Install [Docker](https://docs.docker.com/get-docker/) including [Docker-Compose](https://docs.docker.com/compose/install/).
+2. Create a directory for your local public FullNode, and `cd` into it.
+3. Download the following YAML configuration files:
+
+    - Click on [Public FullNode Docker Compose](https://github.com/aptos-labs/aptos-core/tree/main/docker/compose/public_full_node/docker-compose.yaml) and save the file, or run the below command on your terminal:
+    ```
+    wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/docker/compose/public_full_node/docker-compose.yaml
+    ```
+
+    and
+
+    - Click on [Public FullNode Aptos-core](https://github.com/aptos-labs/aptos-core/tree/main/docker/compose/public_full_node/public_full_node.yaml) and save the file, or run the below command on your terminal:
+    ```
+    wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/docker/compose/public_full_node/public_full_node.yaml
+    ```
+4. The Aptos devnet publishes the `genesis.blob` and `waypoint.txt` files. Download them:
+
+    - Click on [genesis][devnet_genesis] or run the below command on your terminal:
+      ```
+      wget https://devnet.aptoslabs.com/genesis.blob
+      ```
+
+    - Click on [waypoint][devnet_waypoint] and save the file, or run the below command on your terminal:
+      ```
+      wget https://devnet.aptoslabs.com/waypoint.txt
+      ```
+
+5. Start Docker Compose by running the command:
+
+    ```
+    docker-compose up
+    ```
+
+## Verify the correctness of your FullNode
+
+### Verify initial synchronization
+
+During the initial synchronization of your FullNode, there may be a lot of data to transfer. You can monitor the progress by querying the metrics port to see what version your node is currently synced to. Run the following command:
+
+```
+curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type=\"synced\"}"
+```
+
+The command will output the current synced version of your node. For example:
+```
+$ curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type=\"synced\"}"
+aptos_state_sync_version{type="synced"} 71000
+```
+
```

Comment:
This is from the old version of the docs, I recently updated this, since this version here doesn't work for state sync v2. Have this instead:

During the initial synchronization of your FullNode, there may be a lot of data to transfer. Progress can be monitored by querying the metrics port to see what version your node is currently synced to. Run the following command to output the current synced version of your node:
```
curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{.*\"synced\"}" | awk '{print $2}'
```

Example output:
```
71000
```